content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,64 +14,65 @@
 
       <section id="about">
-        <h2>About This Pattern</h2>
-        <p>WAI-ARIA supports two types of <a href="#checkbox" class="role-reference">checkbox</a> widgets: dual-state checkboxes toggle between two choices -- checked and not checked -- and tri-state checkboxes, which  allow an additional third state known as partially checked.</p>
+        <h2>このパターンについて</h2>
+        <p>WAI-ARIA は、2 種類の <a href="#checkbox" class="role-reference">checkbox</a> ウィジェットをサポートしています。二値状態のチェックボックスは、チェック済みと未チェックの 2 つの選択肢を切り替え、3 値状態のチェックボックスは、部分的にチェック済みとして知られる追加の 3 番目の状態を許可します。</p>
         <p>
-          One common use of a tri-state checkbox can be found in software installers where a single tri-state checkbox is used to represent and control the state of an entire group of install options.
-          And, each option in the group can be individually turned on or off with a dual state checkbox.
+          3 値状態チェックボックスの一般的な使用法の 1 つは、ソフトウェアインストーラーにあります。そこでは、単一の 3 値状態チェックボックスを使用して、インストールオプションのグループ全体の状態を表し、制御します。
+          そして、グループ内の各オプションは二値状態チェックボックスで個別にオン又はオフにできます。
         </p>
         <ul>
-          <li>If all options in the group are checked, the overall state is represented by the tri-state checkbox displaying as checked.</li>
-          <li>If some of the options in the group are checked, the overall state is represented with the tri-state checkbox displaying as partially checked.</li>
-          <li>If none of the options in the group are checked, the overall state of the group is represented with the tri-state checkbox displaying as not checked.</li>
+          <li>グループ内のすべてのオプションがチェックされている場合、全体の状態は、チェック済みとして表示される3値状態チェックボックスによって表されます。</li>
+          <li>グループ内の一部のオプションがチェックされている場合、全体の状態は、部分的にチェック済みとして表示される3値状態チェックボックスで表されます。</li>
+          <li>グループ内のどのオプションもチェックされていない場合、グループの全体の状態は、チェックされていないと表示される3値状態チェックボックスで表されます。</li>
         </ul>
-        <p>The user can use the tri-state checkbox to change all options in the group with a single action:</p>
+        <p>利用者は、3 値状態チェックボックスを使用して、単一のアクションでグループ内のすべてのオプションを変更できます。</p>
         <ul>
-          <li>Checking the overall checkbox checks all options in the group.</li>
-          <li>Unchecking the overall checkbox will uncheck all options in the group.</li>
+          <li>全体のチェックボックスをチェックすると、グループ内のすべてのオプションがチェックされます。</li>
+          <li>全体のチェックボックスのチェックを外すと、グループ内のすべてのオプションのチェックが外れます。</li>
           <li>
-            And, In some implementations, the system may remember which options were checked the last time the overall status was partially checked.
-            If this feature is provided, activating the overall checkbox a third time recreates that partially checked state where only some options in the group are checked.
+            そして、一部の実装では、システムは、全体の状態が最後に部分的にチェックされたときにどのオプションがチェックされたかを記憶している場合があります。
+            この機能が提供されている場合、全体のチェックボックスを3回目にアクティブにすると、グループ内の一部のオプションのみがチェックされている部分的にチェックされた状態が再作成されます。
```

Comment:
このactivateも「作動」が妥当？

```suggestion
            この機能が提供されている場合、全体のチェックボックスを3回目に作動すると、グループ内の一部のオプションのみがチェックされている部分的にチェックされた状態が再作成されます。
```

Reply:
[用語集 > activate](https://docs.google.com/spreadsheets/d/1V8wX-pxAO-zuYwTSvTSuZ_FtnV47su6Tyy2vM5GEOLw/edit?gid=0#gid=0&range=51:51) に基づいて「動作させる」ですかね


```suggestion
            この機能が提供されている場合、全体のチェックボックスを3回目に動作させると、グループ内の一部のオプションのみがチェックされている部分的にチェックされた状態が再作成されます。
```